lab-yunjoo/test/storage-test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
```

Comment:
These tests work if I manually go into your test/data directory and mkdir a note directory, but they fail if I don't. That's the reason why travisCI says that your checks have failed.